<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CheckoutUI</class>
 <widget class="QDialog" name="CheckoutUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="SKU_LineEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="Total_Button">
       <property name="text">
        <string>Total</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="PriceOutput_Edit"/>
   </item>
   <item>
    <widget class="QPushButton" name="Clear_Button">
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
